src/pages/Market/Header/index.tsx

```diff
@@ -0,0 +1,72 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import BigNumber from 'bignumber.js';
+import { Paper, Typography } from '@mui/material';
+import { useTranslation } from 'translation';
+import {
+  formatCommaThousandsPeriodDecimal,
+  convertCentsToDollars,
+  formatCentsToReadableValue,
+} from 'utilities/common';
+import { useStyles } from '../styles';
+
+interface IHeaderProps {
+  totalSupplyCents: BigNumber;
+  totalBorrowCents: BigNumber;
+  availableLiquidityCents: BigNumber;
+  totalTreasuryCents: BigNumber;
+}
+
+const Header: React.FC<IHeaderProps> = ({
+  totalSupplyCents,
+  totalBorrowCents,
+  availableLiquidityCents,
+  totalTreasuryCents,
+}) => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+  return (
+    <Paper css={styles.headerRoot}>
+      <div css={styles.row}>
+        <Paper css={styles.box}>
+          <Typography variant="small1" css={styles.title}>
+            {t('market.totalSupply')}
+          </Typography>
+          <Typography variant="h3" css={styles.value}>
+            {formatCentsToReadableValue({ value: totalSupplyCents })}
+          </Typography>
+        </Paper>
+        <Paper css={styles.box}>
+          <Typography variant="small1" css={styles.title}>
+            {t('market.totalBorrow')}
+          </Typography>
+          <Typography variant="h3" css={styles.value}>
+            {formatCommaThousandsPeriodDecimal(convertCentsToDollars(totalBorrowCents.toNumber()))}
+          </Typography>
+        </Paper>
+        <Paper css={styles.box}>
+          <Typography variant="small1" css={styles.title}>
+            {t('market.availableLiquidity')}
+          </Typography>
+          <Typography variant="h3" css={styles.value}>
+            {formatCommaThousandsPeriodDecimal(
+              convertCentsToDollars(availableLiquidityCents.toNumber()),
+            )}
+          </Typography>
+        </Paper>
+        <Paper css={styles.box}>
+          <Typography variant="small1" css={styles.title}>
+            {t('market.totalTreasury')}
+          </Typography>
+          <Typography variant="h3" css={styles.value}>
+            {formatCommaThousandsPeriodDecimal(
+              convertCentsToDollars(totalTreasuryCents.toNumber()),
+            )}
```

Comment:
Sorry I didn't comment again on those, but same thing here they need to use `formatCentsToReadableValue`. Sorry for that, I just didn't want to repeat the comments, I felt it would make it annoying 😬